beluga/include/beluga/sensor/data/occupancy_grid.hpp

```diff
@@ -181,6 +181,13 @@ class BaseOccupancyGrid2 : public BaseLinearGrid2<Derived> {
              return value_traits.is_occupied(value);
            });
   }
+
+  [[nodiscard]] auto unknown_obstacle_data() const {
+    return this->self().data() |
+           ranges::views::transform([value_traits = this->self().value_traits()](const auto& value) {
+             return std::make_tuple(value_traits.is_occupied(value), value_traits.is_unknown(value));
+           });
+  }
```

Comment:
@DPR00 I'm not a huge fan of using data types with logically invalid states (what does `(true, true)` mean?). I'd rather use an `enum` or a fully fledged [`tribool`](https://www.boost.org/doc/libs/1_61_0/doc/html/boost/logic/tribool.html) equivalent. Also, perhaps `ternary_data()` would be more accurate for a name.

Reply:
That's true. I'm going for `enum class`, since it would be more readable.